app/javascript/components/TripFormModal/TripFormPlaces.js

```diff
@@ -84,6 +85,7 @@ const TripFormPlaces = () => {
                 onSelect={(location) => appendPlaces(location)}
                 clearAfterSelect
                 isInvalid={!!errors[FIELD_NAMES.PLACES]}
+                aria-labelledby="places"
```

Comment:
This shouldn't be needed since `controlId={FIELD_NAMES.PLACES}` adds the correct `for` and `id` attributes to the `label` and `input`.